intuita-webview/src/jobDiffView/App.tsx

```diff
@@ -104,9 +115,8 @@ function App() {
 	return (
 		<main className="App">
 			<JobDiffViewContainer
-				scrollIntoHash={scrollIntoHash}
 				diffId={diffId}
-				jobs={data}
+				jobs={[data[jobIndex]!]}
```

Comment:
you can check explicitly to see if it exists to satisfy the compiler (if it doesn't, provide an empty array)